pincer/objects/guild/guild.py

```diff
@@ -482,6 +484,222 @@ async def modify_member(self, _id: int, **kwargs) -> GuildMember:
+            bots, as well as users with the permission manage_messages or manage_channel, are unaffected
+        position : Optional[:class:`int`]
+            sorting position of the channel
+        permission_overwrites : Optional[List[:class:`~pincer.objects.guild.overwrite.Overwrite`]]
+            the channel's permission overwrites
+        parent_id : Optional[:class:`~pincer.utils.snowflake.Snowflake`]
+            id of the parent category for a channel
+        nsfw : Optional[:class:`bool`]
+            whether the channel is nsfw
+
+        """
+        ...
+
+    async def create_channel(self, **kwargs):
+        data = await self._http.post(f"guilds/{self.id}/channels", data=kwargs)
+        return Channel.from_dict(construct_client_dict(self._client, data=data))
+
+    async def modify_channel_positions(
+        self, *channel: Dict[str, Optional[Union[int, bool, Snowflake]]]
+    ):
+        """|coro|
+        Create a new channel object for the guild.
+
+        Parameters
+        ----------
+
+        \\*channel : Dict[str, Optional[Union[int, bool, :class:`~pincer.utils.snowflake.Snowflake`]
+            Keys:
+                - id : :class:`~pincer.utils.snowflake.Snowflake`
+                - position : Optional[:class:`int`]
+                - lock_permissions : Optional[:class:`bool`]
+                - parent_id : Optional[:class:`~pincer.utils.snowflake.Snowflake`]
+
+        """
+
+    async def list_active_threads(self) -> Tuple[
+        List[Union[PublicThread, PrivateThread]], List[GuildMember]]:
+        """|coro|
+        Returns all active threads in the guild, including public and private threads.
+        """
+        data = await self._http.get(f"guilds/{self.id}/threads/active")
+
+        threads = [Channel.from_dict(channel) for channel in data["threads"]]
+        members = [GuildMember.from_dict(member) for member in data["members"]]
+
+        return threads, members
+
+    async def list_guild_members(self, limit: int = 1, after: int = 0):
+        """|coro|
+        Returns a list of guild member objects that are members of the guild.
+
+        Parameters
+        ----------
+        limit : int
+            max number of members to return (1-1000) |default| :data:`1`
+        after : int
+            the highest user id in the previous page |default| :data:`0`
+        """
+
+        return await self._http.get(
+            f"guilds/{self.id}/members?limit={limit}&after={after}"
+        )
+
+    async def search_guild_members(
+        self, query: str,
+        limit: Optional[int] = None
+    ) -> List[GuildMember]:
+        """|coro|
+        Returns a list of guild member objects whose username or nickname starts with a provided string.
+
+        Parameters
+        ----------
+        query : str
+            Query string to match username(s) and nickname(s) against.
+        limit : Optional[int]
+            max number of members to return (1-1000) |default| :data:`1`
+
+        """
+
```

Comment:
Missing return type in the docstring + redundant line
```suggestion
        limit : Optional[int]
            max number of members to return (1-1000) |default| :data:`1`
        """

```